src/main/java/com/jagrosh/vortex/commands/moderation/SlowmodeCmd.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2016 John Grosh (jagrosh).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.jagrosh.vortex.commands.moderation;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.vortex.Vortex;
+import com.jagrosh.vortex.commands.ModCommand;
+import com.jagrosh.vortex.utils.FormatUtil;
+import com.jagrosh.vortex.utils.OtherUtil;
+import net.dv8tion.jda.core.Permission;
+import net.dv8tion.jda.core.entities.TextChannel;
+
+/**
+ * @author Michaili K (mysteriouscursor+git@protonmail.com)
+ */
+public class SlowmodeCmd extends ModCommand
+{
+    final static int MAX_SLOWMODE = 21600;
+
+    public SlowmodeCmd(Vortex vortex)
+    {
+        super(vortex, Permission.MESSAGE_MANAGE);
```

Comment:
Although it is very useful for users with Manage Messages to be able to enable/disable slowmode, I've noticed how Vortex requires the user to have the permissions needed to fulfill the command (like for softban, you still need Ban Members even though you are effectively kicking the user).
Shall I change this to require Manage Channels permission?